discover/portal/articles/XX-segmentation-and-personalization/03-segment-criteria-reference.markdown

```diff
@@ -0,0 +1,112 @@
+# Defining Segmentation Criteria
```

Comment:
Does this file belong here? There's already a folder for this in the proper place (/user).